content/en/developing-pixelfed/intro.md

```diff
@@ -23,6 +24,6 @@ cd pixelfed
 composer install
 php artisan install
 php artisan serve
+```
 
-# Laravel development server started: http://127.0.0.1:8000
-```
\ No newline at end of file
+The development server will start on: [http://localhost:8000](http://localhost:8000)
```

Comment:
this doesn't need to be a link and probably shouldn't be one

Reply:
Thank for the feedback @trwnh. My workflow when working with new projects is to have the documentation up to one side. So I figured a clickable link would be more usable, happy to remove though as it's not a blocker for me.

For my own learning would you be able to expand on your statement, what problems do you see with this being a link?

Reply:
for most people it will be a broken link, and they will see it as a clickable item on the page which adds unnecessary cognitive load. for search engines, a broken link can be a bad signal for seo and prioritization. yes, it useful for development, but only if you set up your dev env. it's not super important either way, in any case